src/apps/api/pagination.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework.pagination import PageNumberPagination
+
+
+class DatasetPagination(PageNumberPagination):
+    page_size = 10
```

Comment:
can we put these straight on the view instead of having to do this subclass thing? If we have to do it this way, can we rename this `BasicPagination` so we can make it more clear it's for re-use?